src/util/VanDerCorput.cpp

```diff
@@ -116,10 +131,8 @@ VanDerCorput::const_iterator VanDerCorput::end() const
 }
 
 //=============================================================================
-VanDerCorput::const_iterator::const_iterator(const VanDerCorput *seq)
-    : mSeq(seq)
-    , mN(-1)
-    , mFinalIter(false)
+VanDerCorput::const_iterator::const_iterator(const VanDerCorput* seq)
+  : mSeq(seq), mN(-1), mFinalIter(false)
```

Comment:
> I really don't like putting an initialization list on one line. I don't see the difference between this initialization list ... this list of assignments...

Maybe the setting is to allow single line if it's possible. I personally prefer this way, but not that strongly. Let me check if we could enforce to list them per line.

> On a slight tangent, I prefer ... because it's easier to re-order lines. I think @psigen strongly prefers the latter, though.

Sorry @mkoval . Here is another one prefers the latter even with the understanding of the convenience of re-ordering. `, mN(-1)` looks not that good in aesthetic. Hm, I don't know how we decide the code style when there are disagreements in preferences each other. :cry: